htdocs/multicurrency/multicurrency_rate.php

```diff
@@ -229,163 +237,124 @@
 
 // Configuration header
 $head = multicurrencyAdminPrepareHead();
-print dol_get_fiche_head($head, 'ratelist', $langs->trans("ModuleSetup"), -1, "multicurrency");
+dol_fiche_head($head, 'ratelist', $langs->trans("ModuleSetup"), -1, "multicurrency");
 
 // ACTION
 
-if ($action != "updateRate") {
+if ($action!= "updateRate" && $action!= "deleteRate" ) {
 	print '<table class="noborder centpercent">';
 	print '<tr class="liste_titre">';
 	print '<td>'.$langs->trans("FormCreateRate").'</td>'."\n";
 	print '</tr></table>';
 
 	$form = new Form($db);
+
 	print '<form action="'.$_SERVER["PHP_SELF"].'" method="post" name="formulaire">';
 	print '<input type="hidden" name="token" value="'.newToken().'">';
 
-	print '<table><tr>';
+	print '<table class="noborder centpercent"><tr>';
 
-	print ' <td>'.$langs->trans('Date').'</td>';
-	print ' <td>';
-	print $form->selectDate($dateinput, 'dateinput');
-	print '</td>';
+	print ' <td>' . $langs->trans('date') . '</td>';
+	print ' <td><input class="minwidth200" name="dateinput" value="" type="date"></td>';
 
-	print '<td> '.$langs->trans('Currency').'</td>';
-	print '<td>'.$form->selectMultiCurrency((GETPOSTISSET('multicurrency_code') ? GETPOST('multicurrency_code', 'alpha') : $multicurrency_code), 'multicurrency_code', 0, " code != '".$conf->currency."'", true).'</td>';
+	print ' <td>' . $langs->trans('Codemulticurrency') . '</td>';
+	print '<td>' . $form->selectMultiCurrency('', 'multicurrency_code', 1, " code != '".$conf->currency."'", true) . '</td>';
```

Comment:
Initial value must restore value posted for the case we send form and there is an error.

It seems this PR contains too many regression and is based on a very old file. I close it so it will be possible to use a new one based on current develop version.